### PR TITLE
[MNOE-146] Display teams admin panel

### PR DESCRIPTION
**Pull request by hedudelgado**

### CODE REVIEW

**Comment by hedudelgado** [comment]
@ouranos

File: api/app/controllers/mno_enterprise/jpi/v1/admin/teams_controller.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      else
+        query = fetch_teams(params[:organization_id])
+        @teams = query.to_a
+        response.headers['X-Total-Count'] = query.meta.record_count
+      end
+    end
+
+    protected
+
+    def fetch_all_teams(organization_id)
```
Comment (x4d3) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/teams_controller.rb]: what is the difference between `fetch_all_teams` and `fetch_teams` ?

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/base_resource_controller.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@ def timestamp
   end
 
   def parent_organization
-    @parent_organization ||= current_user.organizations.to_a.find { |o| o.id.to_s == params[:organization_id].to_s }
+    @parent_organization ||= MnoEnterprise::Organization.includes(:orga_relations).find(params[:organization_id]).first
```
Comment (x4d3) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/base_resource_controller.rb]: replace by `find_one(params[:organization_id], :orga_relations)`

File: api/app/views/mno_enterprise/jpi/v1/admin/teams/_team.json.jbuilder [hunk 3]
```diff
@@ -0,0 +1,20 @@
+@all_apps ||= MnoEnterprise::App.all.to_a
```
Comment (ouranos) [inline api/app/views/mno_enterprise/jpi/v1/admin/teams/_team.json.jbuilder]: Do we really to load all apps for each team here?
Couldn't we load and include team -> app_instances -> app
Especially if it's just to display the logo

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/base_resource_controller.rb [hunk 4]
```diff
@@ -19,7 +19,7 @@ def timestamp
   end
 
   def parent_organization
-    @parent_organization ||= current_user.organizations.to_a.find { |o| o.id.to_s == params[:organization_id].to_s }
+    @parent_organization ||= MnoEnterprise::Organization.find_one(params[:organization_id], :orga_relations)
```
Comment (ouranos) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/base_resource_controller.rb]: I think this helper was not used before and is just a bad copy paste  from the regular jpi controller.
How about removing it and inline the code in the controller that uses it?

File: core/lib/mno_enterprise/concerns/models/app_instance.rb [hunk 5]
```diff
@@ -9,6 +9,7 @@ module MnoEnterprise::Concerns::Models::AppInstance
   included do
     property :created_at, type: :time
     property :updated_at, type: :time
+    property :app_id, type: :string
```
Comment (ouranos) [inline core/lib/mno_enterprise/concerns/models/app_instance.rb]: I thought we were getting rid of this in #560

Comment (x4d3) [reply core/lib/mno_enterprise/concerns/models/app_instance.rb]: we are, but little by little.

**Pull request merged**